### DISCUSSION TITLE
how to return detected face as json result

### DISCUSSION THREAD

**Discussion by bigdahaut** [vladmandic/face-api#74] [Q&A]
I would like to do facial recognition with this module(vladmandic/face-api) by api. I found an example on github using this module. but the latter does not do facial recognition. I would like the name of the recognized face to be returned in json. I can't seem to incorporate this part of face recognition into the example I took from the net. I am a beginner in javascript. Here is the github link of the example in question: https://github.com/kvntzn/face-api

**Comment by vladmandic** [comment]
there are several past issues and discussions where that was talked about - can you search for `LabeledFaceDescriptors`?
